[2025-04-02T02:59:33.992+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-02T02:59:34.001+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: first_dag.get_name manual__2025-04-02T02:59:33.189253+00:00 [queued]>
[2025-04-02T02:59:34.004+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: first_dag.get_name manual__2025-04-02T02:59:33.189253+00:00 [queued]>
[2025-04-02T02:59:34.004+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 6
[2025-04-02T02:59:34.010+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): get_name> on 2025-04-02 02:59:33.189253+00:00
[2025-04-02T02:59:34.016+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=9976) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-02T02:59:34.015+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'first_dag', 'get_name', 'manual__2025-04-02T02:59:33.189253+00:00', '--job-id', '45', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmp7kl9lgm2']
[2025-04-02T02:59:34.017+0000] {standard_task_runner.py:72} INFO - Started process 9977 to run task
[2025-04-02T02:59:34.017+0000] {standard_task_runner.py:105} INFO - Job 45: Subtask get_name
[2025-04-02T02:59:34.040+0000] {task_command.py:467} INFO - Running <TaskInstance: first_dag.get_name manual__2025-04-02T02:59:33.189253+00:00 [running]> on host 7d980c7fabe2
[2025-04-02T02:59:34.083+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='grypton' AIRFLOW_CTX_DAG_ID='first_dag' AIRFLOW_CTX_TASK_ID='get_name' AIRFLOW_CTX_EXECUTION_DATE='2025-04-02T02:59:33.189253+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-02T02:59:33.189253+00:00'
[2025-04-02T02:59:34.083+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-02T02:59:34.110+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-04-02T02:59:34.114+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-02T02:59:34.114+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=first_dag, task_id=get_name, run_id=manual__2025-04-02T02:59:33.189253+00:00, execution_date=20250402T025933, start_date=20250402T025934, end_date=20250402T025934
[2025-04-02T02:59:34.154+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-02T02:59:34.168+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-02T02:59:34.169+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
